content/markdown-viewer.js

```diff
@@ -27,7 +29,62 @@ extensions.markdown = {};
         // we'll recalculate it after the panel is shown.
         return x - (panel.boxObject.width || 120) - 2;
     }
+    
+    /**
+     * Exports DOM document with styles to HTML text
+     * @param {Document} document
+     * @param {string} name (document title)
+     */
+    function exportDocument(document, name) {
+        var s = document.styleSheets,
```

Comment:
Nitpick - you are using a lotof <=2 character variable names here, this makes for code that is hard to read over and it really doesn't gain you any measurable performance.